### DISCUSSION TITLE
[Kavita] Right to left reading mode for pdf files

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2562] [Ideas]
### Idea Description

Posted by Kartoffel
· 11 months ago
· 11 Upvotes
https://feats.kavitareader.com/posts/302/right-to-left-reading-mode-for-pdf-files

Description

I primarily read PDF files that have been created from a manga with some OCR information overlaid on top (Mokuro2Pdf). It would be nice to have a right to left reading mode on the pdf reader, just like the manga/comic viewer. This would allow for spreads to be reversed, thus allowing dual page spreads to be rendered properly.

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by 9783e6** [comment]
I’d love to see this feature! It would be especially useful for people who have manga from Humble Bundle’s Kodansha bundles. The PDFs in those are incredibly high resolution and include selectable text, so it’s not really worth converting them to CBZ. However, without proper right-to-left spread order, the reading experience feels a bit lacking.
